kubernetes/cleanup

```diff
@@ -2,5 +2,11 @@
 
 log()   { echo ">>> $1" ; }
 
+[[ -d manifest-templates ]] || { echo >&2 "Please run this script from within the kubernetes folder"; exit 1; }
```

Comment:
This script is meant to be run by `start` script or manually from the same folder, I don't see why we would need to try to make it smarter than we need for our development purposes (we already document that `start` needs to be executed from within the `velum/kubernetes` folder)... I think we can just keep the script "dumb" and simple...